<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyBindWidget</class>
 <widget class="QWidget" name="PropertyBindWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox {
    border: none;
    border-radius: 3px;
    padding: 2px 3px 2px 3px;
}
QComboBox:hover{
background:#4C4E56;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 12px;

    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(:/res/down.svg);
}
QComboBox QAbstractItemView {
	background: #26282D;
}
QComboBox QAbstractItemView::item {
	height:32px;
}
QComboBox QAbstractItemView::item:selected {
    background: #35363C;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,1,0,0,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cb_call_time">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>进入</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>退出</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>key:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_key">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>value:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_value">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_value_type">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>int</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>bool</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>string</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>double</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>float</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hex</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_remove">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background: transparent;
}
QPushButton:hover{
background: #4C4E56;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/res/delete.svg</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../app.qrc"/>
 </resources>
 <connections/>
</ui>
